ui/shadow_priest/presets.ts

```diff
@@ -237,3 +237,98 @@ export const P2_BIS = {
 		],
 	}),
 };
+export const P3_BIS = {
+	name: 'P3 Initial BIS',
+	tooltip: Tooltips.BASIC_BIS_DISCLAIMER,
+	gear: EquipmentSpec.create({
+		items: [
+			ItemSpec.create({
+				id: 31064, // Hood of Absolution
+				enchant: Enchants.GLYPH_OF_POWER,
+				gems: [
+					Gems.MYSTICAL_SKYFIRE_DIAMOND,
+					Gems.GLOWING_SHADOWSONG_AMETHYST,
+				],
+			}),
+			ItemSpec.create({
+				id: 35319,// Vindicator's Pendant of Subjugation
+				gems: [
+					Gems.RUNED_CRIMSON_SPINEL,
+				],
+			}),
+			ItemSpec.create({
+				id: 31070, // Shoulderpads of Absolution
+				enchant: Enchants.GREATER_INSCRIPTION_OF_DISCIPLINE,
+				gems: [
+					Gems.RUNED_CRIMSON_SPINEL,
+					Gems.RUNED_CRIMSON_SPINEL,
+				],
+			}),
+			ItemSpec.create({
+				id: 32590, // Nethervoid Cloak
+				enchant: Enchants.SUBTLETY,
+			}),
+			ItemSpec.create({
+				id: 31065, // Shroud of Absolution
+				enchant: Enchants.CHEST_EXCEPTIONAL_STATS,
+				gems: [
+					Gems.RUNED_CRIMSON_SPINEL,
+					Gems.RUNED_CRIMSON_SPINEL,
+					Gems.RUNED_CRIMSON_SPINEL,
+				],
+			}),
+			ItemSpec.create({
+				id: 32586, // bracers-of-nimble-thought
+				enchant: Enchants.WRIST_SPELLPOWER,
+			}),
+			ItemSpec.create({
+				id: 31061, // handguards-of-absolution
+				enchant: Enchants.GLOVES_SPELLPOWER,
+				gems: [
+					Gems.RUNED_CRIMSON_SPINEL,
+				],
+			}),
+			ItemSpec.create({
+				id: 32256, // waistwrap-of-infinity
+			}),
+			ItemSpec.create({
+				id: 30916, // /leggings-of-channeled-elements
+				enchant: Enchants.RUNIC_SPELLTHREAD,
+				gems: [
+					Gems.RUNED_CRIMSON_SPINEL,
+					Gems.RUNED_CRIMSON_SPINEL,
+					Gems.RUNED_CRIMSON_SPINEL,
+				],
+			}),
+			ItemSpec.create({
+				id: 32239, // Slippers of the Seacaller
+				enchant: Enchants.BOARS_SPEED,
+				gems: [
+					Gems.RUNED_CRIMSON_SPINEL,
+					Gems.RUNED_CRIMSON_SPINEL,
+				],
+			}),
+			ItemSpec.create({
+				id: 32527, // ring-of-ancient-knowledge
+				enchant: Enchants.RING_SPELLPOWER,
+			}),
+			ItemSpec.create({
+				id: 32527, // ring-of-ancient-knowledge
```

Comment:
If you only want to have 1 BIS list for the phase then it should be a 'realistic bis' list, which I would assume shouldn't use double rings here (maybe the hyjal rep ring instead?). If you want an 'absolute bis' list too that's fine, we have have 2 lists.

Reply:
Double rings is definitely realistic BiS for spriest.